### DISCUSSION TITLE
How to make auto section numbering like the documentation page of myst-parser

### DISCUSSION THREAD

**Discussion by zxtxin** [executablebooks/meta#754] [Q&A]
For example, in 
https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html
this page.
The sub title is auto numbering, but the main title is excluded.
How can I make it?

**Comment by chrisjsewell** [comment]
Yeh so I "cheated" there, and used some custom CSS 😅 https://github.com/executablebooks/MyST-Parser/blob/75ef9cb7b65c98d969724fc6c096e8d6209c5ea0/docs/_static/custom.css#L1
